TypeCobol/Compiler/Symbols/VariableSymbol.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System.Collections.Generic;
+using TypeCobol.Compiler.Scopes;
+
+namespace TypeCobol.Compiler.Symbols
+{
+    /// <summary>
+    /// The Symbol of a Variable declaration
+    /// </summary>
+    public class VariableSymbol : Symbol
+    {
+        /// <summary>
+        /// Named constructor
+        /// </summary>
+        /// <param name="name"></param>
+        public VariableSymbol(string name)
+            : base(name, Kinds.Variable)
+        {
+        }
+
+        /// <summary>
+        /// The Global index associated to this variable.
+        /// </summary>
+        public int GlobalIndex
```

Comment:
What is the use of GlobalIndex ?
I think we discussed that every Symbol will have a unique index.
This index is incremented each time a symbol is created.

If so, the GlobalIndex is also useful on Paragraph/Section to implement issues like #1448. So it could be move on Symbol class.